compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -91,6 +91,13 @@ static llvm::cl::opt<bool> clEnableVectorContractCustomKernels(
                    "LLVMCPUMmt4dVectorLowering pass."),
     llvm::cl::init(false));
 
+static llvm::cl::opt<bool> clForceArmStreaming(
+    "iree-llvmcpu-force-arm-streaming",
+    llvm::cl::desc(
+        "Enables Armv9-A streaming SVE mode for any dispatch region that "
+        "contains scalable vectors. Requires the +sme feature flag."),
+    llvm::cl::init(false));
```

Comment:
It would be good to clarify this meaning of this flag a bit. Let me suggest an updated description:
```
Force IREE (and LLVM) to enable the Armv9-A streaming mode for any dispatch region that contains scalable vectors (i.e. switch from SVE to SSVE). Requires the +sme feature flag."
```

Also, add a comment above, e.g.:
```
By default, IREE does not enable the Armv9-A streaming mode in the presence of scalable vectors (even when using `+sme`). This flag will effectively make IREE/LLVM switch from SVE to SSVE in dispatch regions with scalable vectors. 
```